staging/src/kubevirt.io/api/core/v1/schema_swagger_generated.go

```diff
@@ -394,7 +394,7 @@ func (LaunchSecurity) SwaggerDoc() map[string]string {
 func (SEV) SwaggerDoc() map[string]string {
 	return map[string]string{
 		"policy":      "Guest policy flags as defined in AMD SEV API specification.\nNote: due to security reasons it is not allowed to enable guest debugging. Therefore NoDebug flag is not exposed to users and is always true.",
-		"attestation": "If specified, run the attestation process for a vmi.\n+opitonal",
+		"attestation": "If specified, run the attestation process for a vmi.\n+optional",
```

Comment:
It's interesting that after this change the only thing that gets changed is the description.. Does the `+optional` even effective when `omitempty` is provided?

Reply:
@iholder101 I have no idea how this stuff works O:-)

To my untrained eye it looks like a semantically-meaningful annotation, but after fixing it the generated schema is changed in a way that seems to imply it's just passed through as part of the description. Maybe there's more to it, but that should be investigated by someone who has at least some understanding of the whole mechanism.

Nothing wrong with fixing the obvious typo in the meantime though ;)

Reply:
> Nothing wrong with fixing the obvious typo in the meantime though ;)

Yup! :)
Thank you for the fix! I'm just genuinely curious :)

Maybe @xpivarc would know?

Reply:
The tooling picks up either the optional or omitempty afaik so it is redundant in a sense

Reply:
Thanks @xpivarc.
Should we drop the `+optional`s in a follow-up?

Reply:
This was raised in the past and the reason both exist is because it is dependent on the CRD builder (that translate to the schema). As the project may be processed by other tools on some D/S, the de-facto agreement is to add both per my understanding.

Reply:
Thanks for clarifying @EdDev!